app/js/pluginManager.js

```diff
@@ -60,8 +60,6 @@ function PluginManager() {
 		if (plugin.name === home) {
 			plugin.on('did-finish-load', plugin.show);
 			current = plugin;
-		} else {
-			plugin.on('did-finish-load', plugin.hide);
```

Comment:
Plugins now start out as hidden to avoid logic errors of `plugin.hide()` & `webview.stop()` logic executing for every plugin upon load.